dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/header/HeaderExchangeServer.java

```diff
@@ -128,7 +128,8 @@ public void startClose() {
 
     private void sendChannelReadOnlyEvent() {
         Request request = new Request();
-        request.setEvent(Request.READONLY_EVENT);
+        request.setEvent(true);

```

Comment:
the original `setEvent` is necessary to keep, so we will end up with two `setEvent` methods:

```java
void setEvent(bool);
void setEvent(String);
```